types.go

```diff
@@ -144,7 +160,11 @@ func toFloat(in interface{}) (float64, error) {
 
 	switch inValue.Kind() {
 	case reflect.String:
-		return strconv.ParseFloat(inValue.String(), 64)
+		s := strings.TrimSpace(inValue.String())
+		if s == "" {
+			return 0, nil
+		}
+		return strconv.ParseFloat(s, 64)
```

Comment:
these two cases let an empty column for a float or in be parsed as 0. Is this behavior we want? e.g.

```
col1,col2,col3
3,,5
10,11,12
```
if all fields are ints to be parsed as `3,0,5` and `10,11,12`, which looks like an error to me.